src/main/resources/alfresco/subsystems/Search/solr/solr-search.properties

```diff
@@ -4,3 +4,7 @@ solr.port.ssl=8443
 solr.query.includeGroupsForRoleAdmin=false
 solr.query.maximumResultsFromUnlimitedQuery=${system.acl.maxPermissionChecks}
 solr.baseUrl=/solr
+
+# Http Client Factory Properties
+http.socket.timeout=2000
```

Comment:
Please rename the properties to "solr.http.socket.timeout" and "solr.http.connection.timeout". As those are specifically for solrHttpClientFactory.